atd-vzd/migrations/default/1707751774791_update_lookup_tables_v27/up.sql

```diff
@@ -0,0 +1,531 @@
+update public.atd_txdot__agency_lkp set agency_desc = 'HOUSTON CHRISTIAN UNIVERSITY POLICE DEPARTMENT' where agency_id = 1478;
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3339, 'WALKER COUNTY SHERIFF''S OFFICE');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3340, 'EDGEWOOD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3341, 'SMILEY CITY MARSHAL''S OFFICE');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3342, 'SHEPHERD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3343, 'HEMPSTEAD ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3344, 'HALE CENTER POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3345, 'SALADO INDEPENDENT SCHOOL DISTRICT POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3346, 'WILLS POINT ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3347, 'MAUD ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3348, 'FROST POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3349, 'SCURRY-ROSSER ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3350, 'JARRELL ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3351, 'PORT ARTHUR ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (3352, 'COOPER ISD POLICE DEPARTMENT');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (10221, 'CITY OF MESQUITE');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (10222, 'CITY OF ARLINGTON');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (10223, 'TOM GREEN COUNTY');
+insert into public.atd_txdot__agency_lkp (agency_id, agency_desc) values (10224, 'FORT BEND COUNTY DISTRICT ATTORNEY''S OFFICE');
+insert into public.atd_txdot__autonomous_level_engaged_lkp (autonomous_level_engaged_id, autonomous_level_engaged_desc) values (3, 'CONDITIONAL AUTOMATION');
+insert into public.atd_txdot__autonomous_level_engaged_lkp (autonomous_level_engaged_id, autonomous_level_engaged_desc) values (4, 'HIGH AUTOMATION');
+insert into public.atd_txdot__autonomous_level_engaged_lkp (autonomous_level_engaged_id, autonomous_level_engaged_desc) values (5, 'FULL AUTOMATION');
+update public.atd_txdot__autonomous_level_engaged_lkp set autonomous_level_engaged_desc = 'AUTOMATION LEVEL UNKNOWN' where autonomous_level_engaged_id = 6;
+update public.atd_txdot__drvr_lic_cls_lkp set drvr_lic_cls_desc = 'AUTONOMOUS' where drvr_lic_cls_id = 95;
```

Comment:
Looked at this one randomly, and it is interesting that this value changed from `NOT REPORTED` to `AUTONOMOUS`. Seems like something that could affect reporting if the VZ team ever looked at autonomous vs non-autonomous based on this field. Goes to show what a huge challenge this all is with CRIS changing existing lookups!

Edit: Actually, could you check on the rows that follow this? Just struck me as out of the ordinary that CRIS would update some of these lookup table's `NOT REPORTED` row to `AUTONOMOUS` like ethnicity and injury severity.

Reply:
Yeah i mean im not sure... the script does seem to be working properly bc i checked the actual lkp export and CRIS really did change all these values to `AUTONOMOUS`... for drivers license there are already other lkp values for `OTHER` and `UNKNOWN` so seems like maybe unknown is functioning as their new `NOT REPORTED` for example. same for ethnicity and injury, there are `UNKNOWN` values so i could see how `UNKNOWN` and `NOT REPORTED` are very similar and maybe unnecessary to have both, hence swapping not reported for autonomous. does seem like something we should point out to the VZ team for the things they can see in the UI like injury and ethnicity, thanks for flagging this!!!

Reply:
side note i find it funny that there are autonomous options for people records now.. i would imagine autonomous cars would just not have a person record for the driver lol?

Reply:
+1 - so strange! And, also why wouldn't they add to the table instead of replacing an existing value? I guess that is part of us being consumers of the data and not having a ton of insight into these decisions. Thanks for looking into this! 🙏